scripts/rabbitmq-server

```diff
@@ -123,6 +123,11 @@ else
     RABBIT_SASL_ERROR_LOGGER='{file,"'${RABBITMQ_SASL_LOGS}'"}'
 fi
 
+# Bump ETS table limit to 50000
+if ["x" = "x$ERL_MAX_ETS_TABLES"]; then
```

Comment:
This should be `if [ "x" = "x$ERL_MAX_ETS_TABLES" ]` (note the spaces), otherwise shell will attempt to execute a command. Fixed in `stable`.